added support for managing the packagecloud repo's

mcrauwel opened the pull request:
added packagecloud module as a dependency

tomkrouper commented on metadata.json:
```diff
@@ -0,0 +1,62 @@
+  ],
+  "dependencies": [
+    { "name": "puppetlabs/stdlib", "version_requirement": ">= 3.2.0 <5.0.0" },
+    { "name": "computology/packagecloud", "version_requirement": ">= 0.2.0" }
```
Should this be [computology/computology-packagecloud](https://github.com/computology/computology-packagecloud)?

mcrauwel replied:
i think it needs to be the reference to the puppet forge: https://forge.puppet.com/computology/packagecloud (which links back to the github repo you mentioned)

tomkrouper commented:
@mcrauwel, can you add the assumption to the README.md with the new module required. Also, can you verify that you've tested. (We need to add automated testing on this at some point.) Once that is done, I think we are :+1:

mcrauwel commented:
@tomkrouper I've added the new feature to the README.md with the note for the dependency.

I've also added a basic example file to verify the installation!
